Components/navigation.html

```diff
@@ -0,0 +1,70 @@
+        <aside class = "sidebar flex flex-column gap-l">
+            <a href = "/index.html">Home</a>
+            <p>Components</p>
+            <ul class = "flex flex-column gap-xl">
+                <li><a href="avatar.html">Avatar</a></li>
+                <li><a href="badge.html">Badge</a></li>
+                <li><a href="alert.html" >Alert</a></li>
+                <li><a href="button.html">Button</a></li>
+                <li><a href="card.html">Cards</a></li>
+                <li><a href="input.html">Input</a></li>
+                <li><a href="image.html">Image</a></li>
+                <li><a href="navigation.html" class = "active-link">Navigation</a></li>
+            </ul>
+        </aside>
+
+        <div class = "flex flex-column main gap-m debug">
+            <h1>Navigation with Search Bar</h1>
+
+        <div class = "container flex gap-xl light-border" style = "align-items: center;max-width:100%">
+            <nav class = "navigation flex gap-xl align-center full-width">
+                <div class="nav-brand">Water-Commerce</div>
+                <div class = "input-wrapper flex grow-1 flex-center" >
+                    <input class = "input-field" placeholder="Please Enter Text">
+                </div>
+                <div class = "flex gap-xl flex-center grow-1" >
+                    <button class="btn btn-secondary">Login</button>
+                    <i class="material-icons" >favorite_border</i>
+                    <i class="material-icons" >shopping_cart</i>
+                </div>
+            </nav>
+        </div>
+
+        <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%2859%2C130%2C246%2C1%29&t=seti&wt=none&l=auto&width=503&ds=true&dsyoff=20px&dsblur=68px&wc=false&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=10.5px&lh=142%25&si=false&es=2x&wm=false&code=%253Cnav%2520class%2520%253D%2520%2522navigation%2520flex%2520gap-xl%2520align-center%2520full-width%2522%253E%250A%2520%2509%253Cdiv%2520class%253D%2522nav-brand%2522%253EWater-Commerce%253C%252Fdiv%253E%250A%2520%2509%253Cdiv%2520class%2520%253D%2520%2522input-wrapper%2520flex%2520grow-1%2520flex-center%2522%2520%253E%250A%2520%2520%2520%2520%2509%253Cinput%2520class%2520%253D%2520%2522input-field%2522%2520placeholder%253D%2522Please%2520Enter%2520Text%2522%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%2520%253D%2520%2522flex%2520gap-xl%2520flex-center%2520grow-1%2522%2520%253E%250A%2520%2520%2520%2520%2509%253Cbutton%2520class%253D%2522btn%2520btn-secondary%2522%253ELogin%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2509%253Ci%2520class%253D%2522material-icons%2522%2520%253Efavorite_border%253C%252Fi%253E%250A%2520%2520%2520%2520%2509%253Ci%2520class%253D%2522material-icons%2522%2520%253Eshopping_cart%253C%252Fi%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fnav%253E"
+            style="width: 503px; height: 183px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Don't use inline style here just add these styles in a different class and use that class for all iframes that will be great i think.

Reply:
Understood, will be adding the class before the final submissions
Thanks